presto-hive/src/main/java/com/facebook/presto/hive/OrcFileWriter.java

```diff
@@ -103,6 +105,42 @@ public OrcFileWriter(
                 dwrfEncryptionProvider,
                 dwrfWriterEncryption);
     }
+    public OrcFileWriter(
+            DataSink dataSink,
+            Callable<Void> rollbackAction,
+            OrcEncoding orcEncoding,
+            List<String> columnNames,
+            List<Type> fileColumnTypes,
+            Optional<List<OrcType>> fileColumnOrcTypes,
+            CompressionKind compression,
+            OrcWriterOptions options,
+            int[] fileInputColumnIndexes,
+            Map<String, String> metadata,
+            ZoneId hiveStorageTimeZone,
+            Optional<Supplier<OrcDataSource>> validationInputFactory,
+            OrcWriteValidationMode validationMode,
+            WriterStats stats,
+            DwrfEncryptionProvider dwrfEncryptionProvider,
+            Optional<DwrfWriterEncryption> dwrfWriterEncryption)
+    {
+        this(
+                dataSink,
+                rollbackAction,
+                orcEncoding,
+                columnNames,
+                fileColumnTypes,
+                fileColumnOrcTypes,
+                compression,
+                options,
+                fileInputColumnIndexes,
+                metadata,
+                DateTimeZone.forTimeZone(TimeZone.getTimeZone(ZoneId.of(hiveStorageTimeZone.getId()))),
```

Comment:
It would be cleaner if we remove joda dependency from OrcFileWriter as well.

Reply:
I do agree that ultimately the OrcFileWriter should remove the dependency. I've limited this PR to only the iceberg component of Presto. In order to remove it from the usage you've outlined (orc package) it needs to be removed from a LOT more items i.e. deeper. I'll address those requirements in another PR as it'll be bigger and might require additional testing.